hadoop-hdds/common/pom.xml

```diff
@@ -242,6 +242,10 @@ https://maven.apache.org/xsd/maven-4.0.0.xsd">
       <artifactId>spotbugs-annotations</artifactId>
       <scope>test</scope>
     </dependency>
+      <dependency>
+          <groupId>org.rocksdb</groupId>
+          <artifactId>rocksdbjni</artifactId>
+      </dependency>
```

Comment:
Should remove the corresponding part in `framework/pom.xml` ?

https://github.com/apache/ozone/blob/aea5edfe99ea92687c604378bec4bfbf1773ba80/hadoop-hdds/framework/pom.xml#L112-L115

Reply:
Done.